inlong-sort/sort-connectors/oracle-cdc/src/main/java/org/apache/inlong/sort/cdc/oracle/table/OracleTableSource.java

```diff
@@ -130,13 +129,14 @@ public ScanRuntimeProvider getScanRuntimeProvider(ScanContext scanContext) {
                         .setResultTypeInfo(typeInfo)
                         .setUserDefinedConverterFactory(
                                 OracleDeserializationConverterFactory.instance())
+                        .setMigrateAll(migrateAll)
                         .build();
         OracleSource.Builder<RowData> builder =
                 OracleSource.<RowData>builder()
                         .hostname(hostname)
                         .port(port)
                         .database(database)
-                        .tableList(schemaName + "." + tableName)
+                        .tableList(tableName)
```

Comment:
manager should change the parameter rules noted

Reply:
Manager does not need to change, the previous rule of 'table name' was'<tableName>'. Now it needs to be changed to'<schemaName>.<tableName>`.